core/src/test/java/org/elasticsearch/common/logging/log4j/LoggingConfigurationTests.java

```diff
@@ -151,6 +154,28 @@ public void testResolveConfigInvalidFilename() throws Exception {
         assertThat(logSettings.get("yml"), Matchers.nullValue());
     }
 
+    // tests that config file is not read when we call LogConfigurator.configure(Settings, false)
+    @Test
+    public void testConfigNotRead() throws Exception {
+        Path tmpDir = createTempDir();
+        Path loggingConf = tmpDir.resolve(loggingConfiguration("yaml"));
+        Files.write(loggingConf, "logger.test: INFO, console\n".getBytes(StandardCharsets.UTF_8));
+        Files.write(loggingConf, "appender.console.type: console\n".getBytes(StandardCharsets.UTF_8), StandardOpenOption.APPEND);
```

Comment:
Ooops - that method is new in Java 8 and you'll be backporting this to 2.0 - so you'd need the `StandardCharsets.UTF_8` anyway. I still think its marginally easier to read my way because you don't need `getBytes` and `StandardOpenOptions.APPEND`.